plumber/api/importer.py

```diff
@@ -0,0 +1,573 @@
+        texture_interpolation: str = "Linear",
+        allow_culling: bool = False,
+        editor_materials: bool = False,
+    ):
+        """
+        Initialize the builder.
+
+        Args:
+            file_system: GameFileSystem to use for imports
+            simple_materials: Import simple, exporter-friendly materials
+            texture_format: Texture format ("Png", "Tga")
+            texture_interpolation: Texture interpolation ("Linear", "Closest", "Cubic", "Smart")
+            allow_culling: Enable backface culling
+            editor_materials: Import editor materials instead of invisible ones
+        """
+        self._file_system = file_system
+        self._jobs: List[ImportJob] = []
+
+        # Store material settings to be used for all imports
+        self._material_settings = {
+            "simple_materials": simple_materials,
+            "texture_format": texture_format,
+            "texture_interpolation": texture_interpolation,
+            "allow_culling": allow_culling,
+            "editor_materials": editor_materials,
+        }
+
+    def add_vmf(
+        self, path: str, from_game: bool = True, **options
+    ) -> "ParallelImportBuilder":
+        """
+        Add a VMF import job.
+
+        Args:
+            path: Path to VMF file
+            from_game: Whether to load from game file system
+            **options: VMF import options (see import_vmf for details)
+
+        Returns:
+            Self for method chaining
+        """
+        self._jobs.append(ImportJob(AssetType.VMF, path, from_game, **options))
+        return self
+
+    def add_mdl(
+        self, path: str, from_game: bool = True, **options
+    ) -> "ParallelImportBuilder":
+        """
+        Add an MDL import job.
+
+        Args:
+            path: Path to MDL file
+            from_game: Whether to load from game file system
+            **options: MDL import options (see import_mdl for details)
+
+        Returns:
+            Self for method chaining
+        """
+        self._jobs.append(ImportJob(AssetType.MDL, path, from_game, **options))
+        return self
+
+    def add_vmt(
+        self, path: str, from_game: bool = True, **options
+    ) -> "ParallelImportBuilder":
+        """
+        Add a VMT import job.
+
+        Args:
+            path: Path to VMT file
+            from_game: Whether to load from game file system
+            **options: VMT import options (see import_vmt for details)
+
+        Returns:
+            Self for method chaining
+        """
+        self._jobs.append(ImportJob(AssetType.VMT, path, from_game, **options))
+        return self
+
+    def add_vtf(
+        self, path: str, from_game: bool = True, **options
+    ) -> "ParallelImportBuilder":
+        """
+        Add a VTF import job.
+
+        Args:
+            path: Path to VTF file
+            from_game: Whether to load from game file system
+            **options: VTF import options (see import_vtf for details)
```

Comment:
Also the vtf options need to be the builder-wide material options and rust code needs to be checked too.